seldon/seldon-core-operator/base/seldon-operator-controller-manager-service-svc.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    control-plane: controller-manager
+    controller-tools.k8s.io: "1.0"
+  name: seldon-operator-controller-manager-service
+  namespace: kubeflow
```

Comment:
you can delete `namespace: kubeflow` in all resources, it's set by kfctl

Reply:
Done except for cluster role binding